src/Ranking.elm

```diff
@@ -1,17 +1,105 @@
 module Ranking exposing (Model, Msg, init, update, view)
 
-import Debug exposing (todo)
-import Zipper exposing (Zipper)
+import Browser
+import List.Zipper exposing (current, after)
+import SetZipper exposing (crateZipper)
+import Colors exposing (black, blue, blueGreen, lightBlue, orange, purple, sky, white)
+import Element exposing (centerX, centerY, column, fill, height, html, layout, maximum, padding, rgb255, shrink, spacing, text, width)
+import Element.Background as Background
+import Element.Border as Border
+import Element.Font as Font exposing (center)
+import Element.Input as Input exposing (button)
+import Maybe
+import String
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , view = view >> layout []
+        , update = update
+        }
 
 
 type alias Model =
     { teams : Zipper (Zipper Team)
-    , isShowingInfo : Bool
+    , isShowingInfo : Bool -- What I need to do with this variable?
+    , inPage : Int
+    , windowHeight : Int
     }
 
 
+type Msg
+    = NextPage
+
+
 type alias Team =
     { number : Int
     , name : String
     , score : Float
     }
+
+
+moveZipperBy : Int -> Zipper a -> Zipper a -- Never failed
+moveZipperBy moveBy zipper =
+    let
+        zipperNextCurrent : Maybe (Zipper a)
+        zipperNextCurrent = next zipper
+    in
+        if moveBy == 0 || zipperNextCurrent == Nothing then
+            zipper
```

Comment:
So, in case `zipperNextCurrent == Nothing` the function fails but you don't reflect it to the user.
I think this function has to behave like `List.Zipper.next`.